core/domain/email_jobs_one_off_test.py

```diff
@@ -45,6 +45,19 @@ def test_hashes_get_generated(self):
         # pylint: disable=unused-argument
         def _generate_hash_for_tests(
                 cls, recipient_id, email_subject, email_body):
+            """Generates hash for tests.
+
+            Args:
+                recipient_id: str. ID of the recipient.
+                email_subject: str. Subject of the email.
+                email_body: str. Body of the email.
+                
+            Returns:
+                str. Empty if recipient_id is 'recipient_id2',
+                    None if 'recipient_id1' and 'Email Hash'
+                    otherwise.
```

Comment:
Can we shift `otherwise` to the above line without exceeding the 80 characters limit? If yes, then please go ahead and shift it.

Reply:
Done!